src/main/resources/db/migration/V1_1__create-tables.sql

```diff
@@ -0,0 +1,103 @@
+CREATE TABLE access_token (
+  id    VARCHAR(255) NOT NULL,
+  token VARCHAR(255),
+  PRIMARY KEY (id)
+);
+
+
+CREATE TABLE contributor_license_agreement (
+  id                  BIGINT       NOT NULL AUTO_INCREMENT,
+  corporate_html      LONGTEXT,
+  corporate_markdown  LONGTEXT     NOT NULL,
+  created             DATETIME     NOT NULL,
+  description         VARCHAR(255),
+  individual_html     LONGTEXT,
+  individual_markdown LONGTEXT     NOT NULL,
+  name                VARCHAR(255) NOT NULL,
+  primary_cla         BIT,
+  superseding_cla_id  BIGINT,
+  PRIMARY KEY (id)
+);
+
+
+CREATE TABLE corporate_signature (
+  id                   BIGINT NOT NULL AUTO_INCREMENT,
+  company_name         VARCHAR(255),
+  country              VARCHAR(255),
+  date_of_signature    DATETIME NOT NULL,
+  email                VARCHAR(255),
+  email_domain         VARCHAR(255),
+  git_hub_organization VARCHAR(255),
+  github_login         VARCHAR(255),
+  mailing_address      VARCHAR(255),
+  name                 VARCHAR(255),
+  telephone            VARCHAR(255),
+  title                VARCHAR(255),
+  cla_id               BIGINT NOT NULL,
+  PRIMARY KEY (id)
+);
+
+
+CREATE TABLE individual_signature (
+  id                BIGINT NOT NULL AUTO_INCREMENT,
+  country           VARCHAR(255),
+  date_of_signature DATETIME NOT NULL,
+  email             VARCHAR(255),
+  github_login      VARCHAR(255),
+  mailing_address   VARCHAR(255),
+  name              VARCHAR(255),
+  telephone         VARCHAR(255),
+  cla_id            BIGINT NOT NULL,
+  PRIMARY KEY (id)
+);
+
+
+CREATE TABLE USER (
```

Comment:
recently ADMIn and admin_access_requested were marked as transient. Can we remove these properties?